crates/matrix-sdk-base/src/client.rs

```diff
@@ -385,7 +384,7 @@ impl BaseClient {
                         },
 
                         #[cfg(not(feature = "e2e-encryption"))]
-                        _ => (),
+                        AnySyncTimelineEvent::MessageLike(_) => (),
```

Comment:
Why doing nothing on a `MessageLike` if `e2ee` is disabled? Can't we handle `AnySyncMessageLikeEvent::RoomMessage` (can even be handled at line 344)?

Reply:
All of the `MessageLike` handling code above that's not for redactions is specific to encryption: Trying to decrypt encrypted events, and handling verification events.